File: web/motogp-stats-client/src/app/models/rider.ts
export class Rider {
	private _position: number | string;
	private _name: string;
	private _time: string;
	private _team: string;

	set position(position: number | string) {
		this._position = position;
	}

	get position() {
		return this._position;
	}

	set name(name: string) {
		this._name = name;
	}

	get name() {
		return this._name;
	}

	set time(time: string) {
		this._time = time;
	}

	get time() {
		return this._time;
	}

	set team(team: string) {
		this._team = team;
	}

	get team() {
		return this._team;
	}
}

File: server/motogp-stats-server/src/main/java/com/motogpstats/models/domain/Rider.java
package com.motogpstats.models.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

import static javax.persistence.GenerationType.IDENTITY;

@Setter
@Builder
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "rider")
public class Rider {
	@Getter(onMethod_ = {@Id, @GeneratedValue(strategy = IDENTITY),
			@Column(name = "id")})
	private int id;

	@Getter(onMethod_ = {@Column(name = "name")})
	private String name;

	@Getter(onMethod_ = {@Column(name = "nation")})
	private String nation;

	@Getter(onMethod_ = {@Column(name = "number")})
	private int number;

	@Getter(onMethod_ = {@Column(name = "team")})
	private String team;

	@Getter(onMethod_ = {@Column(name = "bike")})
	private String bike;
}
